File: src/main/java/com/kamikazejam/datastore/mode/profile/store/ProfileStorageLocal.java
package com.kamikazejam.datastore.mode.profile.store;

import com.kamikazejam.datastore.base.storage.StorageLocal;
import com.kamikazejam.datastore.mode.profile.StoreProfile;
import org.jetbrains.annotations.NotNull;

import java.util.UUID;

public class ProfileStorageLocal<X extends StoreProfile> extends StorageLocal<UUID, X> {
    public ProfileStorageLocal() {}

    @Override
    public @NotNull String getLayerName() {
        return "Profile Local";
    }
}

File: src/main/kotlin/com/kamikazejam/datastore/mode/profile/store/ProfileStorageLocal.kt
package com.kamikazejam.datastore.mode.profile.store

import com.kamikazejam.datastore.base.storage.StorageLocal
import com.kamikazejam.datastore.mode.profile.StoreProfile
import java.util.*

class ProfileStorageLocal<X : StoreProfile<X>> : StorageLocal<UUID, X>() {
    override val layerName: String
        get() = "Profile Local"
}
